uae4all_gp2x_0.7.2a/src/memory.cpp

```diff
@@ -989,8 +989,8 @@ static void allocate_memory (void)
         fseek (savestate_file, chip_filepos, SEEK_SET);
         
         void *tmp=malloc(compressed_size);
-        int outSize=allocated_chipmem;
-        int inSize=compressed_size;
+		uLong outSize=allocated_chipmem;
+        uLong inSize=compressed_size;
```

Comment:
I tried first long which worked. But since it goes as parameters into the uncompress method I think uLong is more correct. Did not really understand what uLongf is. Any ideas ?   But t least restoring save states works now...